.dockerignore

```diff
@@ -0,0 +1,13 @@
+# hidden files and directories
+.*
+.*/
```

Comment:
Are we sure we want to always ignore hidden files and folders? 
I would prefer rather using explicit folder and file names and keep the option.

Reply:
Ok, looks good to me :)
Fixed in my last commit